.github/config.yml

```diff
@@ -0,0 +1,11 @@
+# Comment to be posted to on first time issues
+newIssueWelcomeComment: >
+  Thanks for opening your first issue here! Be sure to follow the issue template!
+
+# Comment to be posted to on PRs from first time contributors in your repository
+newPRWelcomeComment: >
+  Thank you for opening this pull request! Please check out our contributing guidelines at https://github.com/pydata/xarray/blob/main/CONTRIBUTING.md .
+
+# Comment to be posted to on pull requests merged by a first time user
+firstPRMergeComment: >
+  Congratulations on merging your first pull request! We here at xarray are proud of you!
```

Comment:
None of this should be here - it's all from your other PR where you are adding the welcome bot.

I notice the branch you created this pull request from is called `main`. To keep different additions to xarray separate from one another you need to use different git branches. Did you remember to follow [this step](https://docs.xarray.dev/en/stable/contributing.html#creating-a-branch) in the contributing guide?

Reply:
Would it be easier if I closed the two pr's that I've made and then open new ones for the same issues and with different git branches?

Reply:
Actually it looks like your other PR is already on a separate branch called `welcome`. I'm not sure what that branch has these changes in it then - you might be merging from `main` into `welcome`. 

For this branch you could either:
a) Create a new branch called `reshaping_intro` and a new PR for that branch, then close this PR,
b) Remove the change associated with the bot PR from this branch, get this branch merged, and just remember to keep everything separate next time.